src/CloudinaryAdapter.php

```diff
@@ -22,10 +22,18 @@
  */
 class CloudinaryAdapter implements FilesystemAdapter
 {
-
-    /** Cloudinary\Cloudinary */
+    /**
+     * @var Cloudinary
+     */
     protected Cloudinary $cloudinary;
 
+    /**
+     * The media resource extensions supported by cloudinary.
+     */
+    public $mediaExtensions = [
```

Comment:
Cloudinary also supports `avif`, so i think you can add that here.

Reply:
here's a list of all of the supported formats https://cloudinary.com/documentation/image_transformations#supported_image_formats